node-api-service/package.json

```diff
@@ -6,22 +6,26 @@
   "license": "AGPL-3.0",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js",
-    "test": "./node_modules/mocha/bin/mocha test/*.js"
+    "start": "nodemon server.js",
+    "test": "./node_modules/mocha/bin/mocha test/*.js",
+    "test:watch": "./node_modules/mocha/bin/mocha --watch test/*.js"
   },
   "dependencies": {
     "amqplib": "^0.5.3",
     "async-retry": "^1.2.3",
     "bignumber.js": "^8.1.1",
     "blake2s-js": "^1.2.2",
+    "boltwall": "^5.0.5",
     "chainpoint-parse": "^4.0.0",
     "envalid": "^5.0.0",
     "eslint-plugin-prettier": "^3.0.1",
-    "ioredis": "^4.9.0",
     "ip": "^1.1.5",
-    "lnrpc-node-client": "^1.1.0",
+    "lnrpc-node-client": "^1.1.2",
     "lodash": "^4.17.4",
+    "lsat-js": "^1.0.5",
+    "macaroons.js": "^0.3.8",
```

Comment:
These two are needed now for the manual lsat creation that embeds the hodl invoice preimage.